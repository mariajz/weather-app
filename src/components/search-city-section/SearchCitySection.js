import React from 'react';
import SearchInput from '../../commons/search-input';

const SearchCitySection = () => {
    return <SearchInput placeholder="Search your city" />;
};

export default SearchCitySection;
