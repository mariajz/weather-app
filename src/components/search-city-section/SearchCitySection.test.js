import { render } from '@testing-library/react-native';
import React from 'react';
import { View as MockView } from 'react-native';
import SearchCitySection from './SearchCitySection';

jest.mock('../search-input', () => {
    return {
        __esModule: true,
        default: () => <MockView />,
    };
});
jest.mock('../../hooks/useGetAllLocations', () => () => ({
    handleFetchLocations: jest.fn(),
}));

describe('SearchCitySection', () => {
    it('should render SearchCitySection', () => {
        const container = render(<SearchCitySection />);

        expect(container).toMatchSnapshot();
    });
});
