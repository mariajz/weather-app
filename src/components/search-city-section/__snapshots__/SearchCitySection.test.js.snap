// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchCitySection should render SearchCitySection 1`] = `<View />`;
